duckduckgo/Android
# Remove string for fire process name from non-default languages

Task/Issue URL: https://app.asana.com/0/414730916066338/1181413749383569
Tech Design URL: 
CC: 

**Description**:
We need to ensure `FireActivity` runs in a separate process

**Steps to test this PR**:
To help test this, add a log statement to `FireActivity line 86`: 
`Timber.v("comparing process: ${it.pid} - ${it.processName}")`

1. Launch the app and tap the fire button
1. Verify that log statement appears, and shows the process name ending with `:fire`
1. Repeat using the automatic data clearer, as per testing steps outlined in https://github.com/duckduckgo/Android/pull/858


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
